images/win/scripts/Installers/Install-AWS-SAM.ps1

```diff
@@ -0,0 +1,31 @@
+################################################################################
+##  File:  Install-AWS-SAM.ps1
+##  Desc:  Install aws sam cli
+##         https://aws.amazon.com/serverless/sam/
+################################################################################
+
+choco install aws-sam-cli -y
+Install-MSI -MsiUrl "https://github.com/awslabs/aws-sam-cli/releases/latest/download/AWS_SAM_CLI_64_PY3.msi" -MsiName "AWS_SAM_CLI_64_PY3.msi"
+
+$command = Get-Command -Name 'sam'
+
+if ($command)
+{
+    Write-Host "aws sam cli on path"
```

Comment:
Please, use uppercase for tool name: `AWS SAM CLI on path` and `AWS SAM CLI is not on path`